### PR TITLE
[EPMEDU-815]: Log out from my account

### PR DESCRIPTION
**Pull request by Makentoshe**
**Account Screen changes:**
- Implement logout cancelation dialog appears on logout button click
- Add AccountViewModel
- Remove MoreViewModel due to lack of functionality

<details>
<summary>Videos:</summary>
<video src="https://user-images.githubusercontent.com/20960889/189093432-b35e3d22-d710-4d81-a0b2-c18daf611acc.mp4"/>
</details>

### CODE REVIEW

**Comment by Makentoshe** [comment]
Should I reuse MoreViewModel for the account screen? Or should I create a new ViewModel?
I removed nothing from MoreScreen right now until it will be clear.

**Comment by egorikftp** [comment]
@Makentoshe Please fix conflicts

**Comment by egorikftp** [comment]
> Should I reuse MoreViewModel for the account screen? Or should I create a new ViewModel? I removed nothing from MoreScreen right now until it will be clear.

Better to create separate VM for account screen

File: feature/more/src/main/java/com/epmedu/animeal/more/AccountViewModel.kt [hunk 1]
```diff
@@ -5,13 +5,13 @@ import androidx.lifecycle.viewModelScope
...
 import com.epmedu.animeal.common.presentation.viewmodel.delegate.EventDelegate
-import com.epmedu.animeal.more.MoreViewModel.Event
+import com.epmedu.animeal.more.AccountViewModel.Event
 import dagger.hilt.android.lifecycle.HiltViewModel
 import kotlinx.coroutines.launch
 import javax.inject.Inject
 
 @HiltViewModel
-internal class MoreViewModel @Inject constructor(
+internal class AccountViewModel @Inject constructor(
```
Comment (egorikftp) [inline feature/more/src/main/java/com/epmedu/animeal/more/AccountViewModel.kt]: maybe better to move into `account` package

File: feature/more/src/main/java/com/epmedu/animeal/more/account/AccountViewModel.kt [hunk 2]
```diff
@@ -1,17 +1,17 @@
-package com.epmedu.animeal.more
```
Comment (Fileek) [inline feature/more/src/main/java/com/epmedu/animeal/more/account/AccountViewModel.kt]: Please create viewmodel package and move AccountViewModel there

File: feature/more/src/main/java/com/epmedu/animeal/more/account/AccountScreenUI.kt [hunk 3]
```diff
@@ -20,12 +26,28 @@ import com.epmedu.animeal.foundation.button.AnimealTextButtonWithIcon
...
+            onConfirm = {
+                showLogoutCancellationAlert = false
+                onLogout()
+                Toast.makeText(context, R.string.profile_logout_success, Toast.LENGTH_LONG).show()
+            }
+        )
+    }
+
     Scaffold(
         modifier = Modifier.fillMaxSize().statusBarsPadding(),
```
Comment (Fileek) [inline feature/more/src/main/java/com/epmedu/animeal/more/account/AccountScreenUI.kt]: Minor: please put modifier calls on separate lines when there is more than one call

**Review by Fileek — APPROVED** [review]

**Pull request merged**